.github/workflows/reusable_trivy-scans.yml

```diff
@@ -1,51 +1,39 @@
 ---
 name: Run Trivy Scans
+
 on:
-    workflow_call:
-jobs:
-    build-image:
-        runs-on: ubuntu-latest
-        steps:
-            - name: Check out GitHub Repo
-              uses: actions/checkout@v4
+  workflow_call:
 
-            - name: Build an image from Dockerfile
-              run: |
-                  git config --global --add safe.directory $GITHUB_WORKSPACE;
```

Comment:
This isn't needed any more?

Reply:
It was never needed as far as I can tell, as no further git operations are done after the checkout step.

Reply:
:+1: